src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -335,31 +329,36 @@ private static void loadSwitchAndItemUseMaterialsLists() {
 		for (String matName : items) {
 			if (ItemLists.GROUPS.contains(matName)) {
 				List<String> group = ItemLists.getGrouping(matName);
-				ItemUseMaterials.addAll(group);
+				if (group != null && !group.isEmpty())
+					ItemUseMaterials.addAll(group);
 			} else {
 				ItemUseMaterials.add(matName);
 			}
 		}
 	}
 	
 	public static void sendError(String msg) {
-
-		Towny.getPlugin().getLogger().warning("Error could not read " + msg);
+		Towny.getPlugin().getLogger().warning(() -> String.format("Error could not read %s",msg));
 	}
 	
 	public static SpawnLevel getSpawnLevel(ConfigNodes node)
 	{
-		SpawnLevel level = SpawnLevel.valueOf(config.getString(node.getRoot()).toUpperCase());
-		if(level == null) {
-			level = SpawnLevel.valueOf(node.getDefault().toUpperCase());
+		String configString = config.getString(node.getRoot());
+		SpawnLevel spawnLevel;
+		if (configString != null) {
+			spawnLevel = SpawnLevel.valueOf(configString.toUpperCase());
+		} else {
+			spawnLevel = SpawnLevel.valueOf(node.getDefault().toUpperCase());
 		}
-		return level;
+		return spawnLevel;
 	}
 	
-	public static NSpawnLevel getNSpawnLevel(ConfigNodes node)
-	{
-		NSpawnLevel level = NSpawnLevel.valueOf(config.getString(node.getRoot()).toUpperCase());
-		if(level == null) {
+	public static NSpawnLevel getNSpawnLevel(ConfigNodes node) {
+		String configString = config.getString(node.getRoot());
+		NSpawnLevel level;
+		if (configString != null) {
+			level = NSpawnLevel.valueOf(configString.toUpperCase());
```

Comment:
Learned recently we need to have Locale.ROOT in the `.toUpperCase(Locale.ROOT)` otherwise turkish servers end up turning i's into strange symbols.

Reply:
Good to know, I'll address it when I get off of my graveyard shift.